### PR TITLE
Fixes #33109 - Date used by katello-certs-check should be formatted c…

### PR DESCRIPTION
**Pull request by BirkirFreyr**
…orrectly by program

2nd try at this, clean edit - somehow completely messed up my previous git and some sleep deprivation errors...

https://github.com/theforeman/foreman-installer/pull/728

### CODE REVIEW

**Comment by ekohl** [comment]
ok to test

**Review by ekohl — APPROVED** [review]

File: bin/katello-certs-check [hunk 1]
```diff
@@ -95,10 +95,9 @@ function check-server-cert-encoding () {
 }
 
 function check-expiration () {
-    DATE=$(date -u +"%b %-d %R:%S %Y")
     CERT_EXP=$(openssl x509 -noout -enddate -in $CERT_FILE | sed -e 's/notAfter=//' | awk '{$NF="";}1')
     CA_EXP=$(openssl x509 -noout -enddate -in $CA_BUNDLE_FILE | sed -e 's/notAfter=//' | awk '{$NF="";}1')
-    DATE_TODAY=$(date -d"${DATE}" +%Y%m%d%H%M%S)
+    DATE_TODAY=$(date +%Y%m%d%H%M%S)
```
Comment (evgeni) [inline bin/katello-certs-check]:
```suggestion
    DATE_TODAY=$(date -u +%Y%m%d%H%M%S)
```

the old command used `date -u` and then formatting that, and your new one doesn't use `-u`, so it's off by whatever UTC offset your timezone has ;)

**Review by evgeni — APPROVED** [review]

File: bin/katello-certs-check (hunk 1 above)
Comment (BirkirFreyr) [reply bin/katello-certs-check]: You are absolutely correct, didn't catch this myself as I'm on UTC myself - committed your fix.

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**